scripts/test/compactionPerformance/system-test-instance.properties

```diff
@@ -168,59 +90,14 @@ sleeper.compaction.files.batch.size=11
 # Valid values are 'arraylist' and 'arrow'
 sleeper.ingest.record.batch.type=arrow
 
-# The way in which partition files are written to the main Sleeper store
-# Valid values are 'direct' (which writes using the s3a Hadoop file system) and 'async' (which writes locally and then
-# copies the completed Parquet file asynchronously into S3)
-sleeper.ingest.partition.file.writer.type=async
-
-# The maximum number of records that the ingest process writes to the local file (arraylist only)
-sleeper.ingest.max.local.records=100000000
-
-# The maximum number of records that the ingest process loads into memory at one time (arraylist only)
-sleeper.ingest.memory.max.batch.size=1000000
-
-# The number of bytes to allocate to the Arrow working buffer, which is used for sorting and other sundry activities (arrow only) [256MB]
-sleeper.ingest.arrow.working.buffer.bytes=268435456
-
-# The number of bytes to allocate to the Arrow batch buffer, which is used to hold the records (arrow only) [1GB]
-sleeper.ingest.arrow.batch.buffer.bytes=1073741824
-
-# The maximum number of bytes to store on the local disk before uploading to the main Sleeper store (arrow only) [2GB]
-sleeper.ingest.arrow.max.local.store.bytes=2147483648
```

Comment:
It looks like these 3 properties were missed when setting the Arrow settings explicitly.